parent/pom.xml

```diff
@@ -208,9 +208,9 @@
                 <version>${swagger.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.eclipse.jetty.orbit</groupId>
-                <artifactId>javax.servlet</artifactId>
-                <version>${jetty-orbit-javax-servlet.version}</version>
+                <groupId>javax.servlet</groupId>
+                <artifactId>javax.servlet-api</artifactId>
+                <version>${javax-servlet.version}</version>
```

Comment:
* If Brooklyn is using classes from the package then the convention is to explicitly declare it as a dependency in the corresponding module(s)
* If it's only a transitive dependency then its version shouldn't be fixed, unless there's a version conflict as this might lead to unexpected behaviour in future where another version is expected.